storage/gcp/gcp.go

```diff
@@ -647,6 +695,21 @@ func (s *spannerSequencer) consumeEntries(ctx context.Context, limit uint64, f c
 	return didWork, nil
 }
 
+// currentTree returns the size and root hash of the currently integrated tree.
+func (s *spannerSequencer) currentTree(ctx context.Context) (uint64, []byte, error) {
+	row, err := s.dbPool.Single().ReadRow(ctx, "IntCoord", spanner.Key{0}, []string{"seq", "rootHash"})
```

Comment:
More of an observation as stream of consciousness rather than a request for change.

I haven't looked through this code before, but it seems funky to me that the `IntCoord` table (I inferred this is for integration, not integers) is accessed via the sequencer. It looks like that ship has sailed and it would take me some more time to work out what the difference between `IntCoord` and `SeqCoord` is. I'm sure there are good reasons for it, but as a first-timer here my intuition was that sequencers would largely play around with `SeqCoord` and there would be some integrator objects that played around with `IntCoord`, with someone having the task of coordinating both of the coordinator tables. As it is, it appears the sequencer runs the whole show.

Reply:
This is only because, currently, this is a synchronous integrating storage and so there isn't really the separation that's present in say `gcp`. This could easily be changed in the future, though.